### ISSUE TITLE
python meterp on Windows 10 returns incorrect operating system

### ISSUE THREAD

**Issue by OJ** [rapid7/metasploit-payloads#152]
Just ran this with a python meterp on my Windows 10 VM:
```
meterpreter > sysinfo
Computer     : DESKTOP-TF4F7AM
OS           : Windows 8 6.2.9200
Architecture : x64
Meterpreter  : python/windows
```
The `OS` here is clearly wrong, so we should probably fix that. The same machine running native meterp shows:
```
meterpreter > sysinfo
Computer        : DESKTOP-TF4F7AM
OS              : Windows 10 (Build 14393).
Architecture    : x64
System Language : en_US
Domain          : WORKGROUP
Logged On Users : 2
Meterpreter     : x64/windows
```

/cc @zeroSteiner.

**Comment by zeroSteiner** [comment]
So I'm having some trouble reproducing this issue. This is my initial attempt with Windows 10 x64, and a fresh install of Python 2.7.12 x64:

```
msf (S:0 J:1) exploit(handler) > exploit -j
[*] Exploit running as background job.


[*] [2016.12.09-17:35:22] Started reverse TCP handler on 172.20.220.134:4444 
[*] [2016.12.09-17:35:22] Starting the payload handler...
[*] [2016.12.09-17:35:24] Sending stage (38623 bytes) to 172.20.220.159
[*] Meterpreter session 1 opened (172.20.220.134:4444 -> 172.20.220.159:50065) at 2016-12-09 17:35:25 -0500
msf (S:0 J:2) exploit(handler) > sessions -i 1
[*] Starting interaction with 1...


meterpreter > sysinfo
Computer     : DESKTOP-LDICAD0
OS           : Windows 10 10.0.10586
Architecture : x64
Meterpreter  : python/windows

meterpreter > 
```

Do you happen to know what version of Python was running on this system? Additionally if it's available, the output of: ```python.exe -c "import platform; print(platform.uname())"``` would help me to track down the issue.

**Comment by OJ** [comment]
Yeah the version of python was 2.7.6. I know it's an older version, but I still see it appear quite a bit.

It might seem like overkill, but do you think we should use ctypes and query the system information like we do with Native meterp in the case of windows?

**Comment by zeroSteiner** [comment]
Yeah, I think that would make sense to be consistent in the output. Ctypes isn't available on versions of Python < 2.5 which is why it wasn't used originally but it should be easy for me to have it use ctypes when it's available.

**Comment by zeroSteiner** [comment]
Closing this because the PR addressing the issue was landed and I'm pretty sure I forgot to close this out. If I'm wrong and the issue persists, just let me know.
